web/src/css_variables.ts

```diff
@@ -37,6 +37,7 @@ export const media_breakpoints = {
     cb4_min: cb4 + "px",
     cb5_min: cb5 + "px",
     short_navbar_cutoff_height: "600px",
+    modal_sidebar_collapse_breakpoint: 1000 / 14 + "em",
```

Comment:
Can we leave a comment describing where the `1000 / 14` values are originating from?

Reply:
Added "// 1000px at legacy 14px font-size, scaled with em to user-chosen font-size"